Sources/Proxy/Representations/SlicedGeometryRepresentationProxy/index.js

```diff
@@ -0,0 +1,142 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
+import vtkCutter from 'vtk.js/Sources/Filters/Core/Cutter';
+import vtkImageMapper from 'vtk.js/Sources/Rendering/Core/ImageMapper';
+import vtkMapper from 'vtk.js/Sources/Rendering/Core/Mapper';
+import vtkPlane from 'vtk.js/Sources/Common/DataModel/Plane';
+
+import vtkAbstractRepresentationProxy from 'vtk.js/Sources/Proxy/Core/AbstractRepresentationProxy';
+
+// ----------------------------------------------------------------------------
+// vtkSlicedGeometryRepresentationProxy methods
+// ----------------------------------------------------------------------------
+
+function vtkSlicedGeometryRepresentationProxy(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkSlicedGeometryRepresentationProxy');
+
+  // Internals
+  model.plane = vtkPlane.newInstance();
+  model.cutter = vtkCutter.newInstance();
+  model.cutter.setCutFunction(model.plane);
+  model.mapper = vtkMapper.newInstance({
+    interpolateScalarsBeforeMapping: false,
+    useLookupTableScalarRange: false,
+    scalarVisibility: false,
+  });
```

Comment:
Are those good defaults for everyone?

Reply:
Great question, I do not know what those really are for. I'll take any suggestions on that.

Reply:
I would not set them if they are not chosen on purpose.